<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNuevoArticuloAlmacen</class>
 <widget class="QDialog" name="DialogNuevoArticuloAlmacen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>333</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>769</width>
    <height>333</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>769</width>
    <height>333</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>82</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>82</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Nuevo Artículo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>159</horstretch>
               <verstretch>160</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>170</width>
               <height>160</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>170</width>
               <height>160</height>
              </size>
             </property>
             <property name="title">
              <string>Imagen</string>
             </property>
             <widget class="QLabel" name="lbImagen">
              <property name="geometry">
               <rect>
                <x>4</x>
                <y>20</y>
                <width>161</width>
                <height>131</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
             </widget>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="pbExaminar">
               <property name="text">
                <string>Examinar...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbCamara">
               <property name="text">
                <string>Cámara</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>563</width>
             <height>121</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>563</width>
             <height>121</height>
            </size>
           </property>
           <property name="title">
            <string>Nuevo Artículo</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QLabel" name="label_8">
                      <property name="text">
                       <string>Código EAN</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="leCodEAN">
                      <property name="maxLength">
                       <number>20</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_9">
                    <item>
                     <widget class="QLabel" name="label_9">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>20</width>
                        <height>13</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>20</width>
                        <height>13</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Tipo</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="cbTipoProducto">
                      <property name="minimumSize">
                       <size>
                        <width>170</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>69</width>
                        <height>16777215</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_8">
                    <item>
                     <widget class="QLabel" name="label_10">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>61</width>
                        <height>13</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>61</width>
                        <height>13</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Stock Mínimo</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="sbMinimo">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>61</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>61</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>8</pointsize>
                       </font>
                      </property>
                      <property name="maximum">
                       <number>1000000000</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>Descripción</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="leDescripcion">
                      <property name="text">
                       <string/>
                      </property>
                      <property name="maxLength">
                       <number>200</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QLabel" name="label_4">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>17</width>
                        <height>16</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>17</width>
                        <height>16</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>IVA</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="leIVA">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>36</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>36</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="maxLength">
                       <number>3</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <item>
                     <widget class="QLabel" name="label_11">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>63</width>
                        <height>22</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>63</width>
                        <height>22</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Stock Óptimo</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="sbOptimo">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>61</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>61</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>8</pointsize>
                       </font>
                      </property>
                      <property name="maximum">
                       <number>1000000000</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_15">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>55</width>
                    <height>16</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>55</width>
                    <height>16</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Comentario</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextEdit" name="teComentario">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>11111111</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="html">
                   <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QPushButton" name="pbCancelar">
           <property name="minimumSize">
            <size>
             <width>121</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>121</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Cancelar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbGuardar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>121</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>121</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Guardar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leCodEAN</tabstop>
  <tabstop>cbTipoProducto</tabstop>
  <tabstop>sbMinimo</tabstop>
  <tabstop>leDescripcion</tabstop>
  <tabstop>leIVA</tabstop>
  <tabstop>sbOptimo</tabstop>
  <tabstop>teComentario</tabstop>
  <tabstop>pbExaminar</tabstop>
  <tabstop>pbCamara</tabstop>
  <tabstop>pbCancelar</tabstop>
  <tabstop>pbGuardar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
